### PR TITLE
Rename --home argument to --resources

### PR DESCRIPTION
**Pull request by nre-ableton**
We don't use anything else inside of the home directory. We also have
gone to the trouble to create an empty "resources" folder in the
repository with a README and .gitignore, which was previously not used
by default.

### CODE REVIEW

**Comment by ala-ableton** [comment]
@nre-ableton I'm interested in reviewing this PR, thanks!

**Comment by nre-ableton** [comment]
@ala-ableton Well, you were the original reviewer picked by the bot, and I'm sure Dana wouldn't mind one fewer review. :smile:

File: run_codenarc.py [hunk 1]
```diff
@@ -200,9 +200,9 @@ def parse_args(args):
     )
 
     arg_parser.add_argument(
-        '--home',
-        default=os.path.realpath(os.path.dirname(__file__)),
-        help='Groovylint home directory',
+        '--resources',
```
Comment (ala-ableton) [inline run_codenarc.py:203]: This is an API breaking change. It requires a version bump.
Please also adapt `README.md:24` and `resources/README.txt:2`.

**Review by ala-ableton — CHANGES_REQUESTED** [review]

File: run_codenarc.py [hunk 2]
```diff
@@ -32,12 +32,12 @@ def __init__(self, num_violations):
 def _build_classpath(args):
     """Construct the classpath from the Groovy/Groovylint homes and JAR versions."""
```
Comment (ala-ableton) [inline run_codenarc.py:33]: Maybe this docstring should also be reworded. Sorry for missing that during my previous review.

Comment (nre-ableton) [reply run_codenarc.py]: No problem! See fc16c7e, which I will move to the front of this branch during rebase.

**Review by ala-ableton — APPROVED** [review]
Once rebased, LGTM 👍🏻

**Comment by nre-ableton** [comment]
@ablbot autosquash

**Pull request merged**